packages/Sortable/stories/Sortable.stories.helpers.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import styled from "styled-components";
+import Sortable from "../src";
+import Input from "../../Input/stories/examples/InputExample";
+import Select from "../../Select/stories/examples/SelectExample";
+
+export const storyStyles = `
+  max-width: 360px;
+`;
+
+const hipsums = [
+  "Small batch raw denim",
+  "Lorem hipsum",
+  "Humblebrag narwhal",
+  "Gluten-free scenester",
+  "Farm-to-table beard oil",
+  "Paleo forage",
+  "Craft banjo",
+  "Sustainable succulents",
+];
+
+const selectOptions = [
+  <option key="1" value="Dr">
+    Dr
+  </option>,
+  <option key="2" value="Mr">
+    Mr
+  </option>,
+  <option key="3" value="Ms">
+    Ms
+  </option>,
+];
+
+const MultiBox = styled.div`
+  display: flex;
+  flex-grow: 1;
+
+  > div + div {
+    margin-left: 8px;
+  }
+
+  select {
+    min-width: 4em;
+  }
+
+  .form-input {
+    width: 100%;
+  }
+`;
+
+export function basicChildren(numChildren) {
+  const children = [];
+
+  for (let index = 1; index <= numChildren; index++) {
+    children.push(
+      <Sortable.Item key={index} sortId={index}>
```

Comment:
It'd be nice if we could just reuse `key` for `sortId` but apparently not possible as we won't have access to it in the parent component